packages/nimble-components/src/wafer-map/index.ts

```diff
@@ -279,6 +287,18 @@ export class WaferMap<
         return this.diesTable !== undefined;
     }
 
+    public async createWorker(): Promise<void> {
```

Comment:
I'm guessing these are internal too, if so could you mark them with the same tag as other internal functions?

Reply:
Yes, they are meant to be used only inside wafer-map component, I modified them
![image](https://github.com/ni/nimble/assets/110180309/b51a27fd-992e-42e9-a4c2-aa87a9f25dc7)